### ISSUE TITLE
Incoming transactions are shown as outgoing and vice versa

### ISSUE THREAD

**Issue by vporton** [fuseio/FuseWebsite#1]
**Describe the bug**
Incoming transactions are shown as outgoing and vice versa:

https://explorer.fuse.io/address/0xaacbaab8571cbeceb46ba85b5981efdb8928545e/transactions
shows many incoming transactions but those should be outgoing (and exchange the sender and receiver addresses).

**Comment by leonprou** [comment]
Hey @vporton,

I don't have a deep knowledge of the Good dollar flow, but why you think they should be outgoing?
The blockscout explorer is used in production for years, and I don't think a crucial bug like this is possible at that stage. 

So if you could give a specific transaction and tell why it should be reversed.

**Comment by vporton** [comment]
@leonprou My GoodDollar account: `0xcb6B592Cb752e8eD9F2c540110c29a069ca9ae4e`.

It has only incoming transactions, but
https://explorer.fuse.io/address/0xcb6b592cb752e8ed9f2c540110c29a069ca9ae4e/transactions
shows them as outgoing transactions from my account.

**Comment by leonprou** [comment]
@vporton oh I think I've got it.
 
the explorer here: https://explorer.fuse.io/address/0xcb6b592cb752e8ed9f2c540110c29a069ca9ae4e/transactions shows that your account calls the contract: `0xaacbaab8571cbeceb46ba85b5981efdb8928545e`. But if you open one of the transactions like https://explorer.fuse.io/tx/0x95bf6a1856faad9cb8573ae716d0dc16ccff6158af609b01ad61027bb78f6362/token_transfers you see that actually you're getting the G$ tokens in the tx.
So basically your accounts requesting funds from the `0xaacbaab8571cbeceb46ba85b5981efdb8928545e` contract
